env/AWSPW.env

```diff
@@ -54,7 +54,7 @@ elif [[ "${step}" = "prep_emissions" ]]; then
 
 elif [[ "${step}" = "waveinit" ]] || [[ "${step}" = "waveprep" ]] || [[ "${step}" = "wavepostsbs" ]] || [[ "${step}" = "wavepostbndpnt" ]] || [[ "${step}" = "wavepostbndpntbll" ]] || [[ "${step}" = "wavepostpnt" ]]; then
 
-    export CFP_MP="YES"
+    export USE_CFP="YES"
```

Comment:
`CFP_MP` was originally added to set a sub-switch of `USE_CFP`. Has the need gone away, particularly with this PR? Has this been tested with PBS `cfp` and Slurm `--multi-prog`?

Reply:
`CFP_MP` and `USE_CPF` are used inconsistently across the workflow.  I think Walter wanted us to move towards using `USE_CFP` consistently.  
The `sbs` scripts were tested with and without CFP.  They call `run_mpmd.sh`.  The point jobs do not work the `CFP_MP=YES` in `develop` on WCOSS2.  I tried, but gave up since those jobs are getting updated.

Reply:
Got it...I agree with consolidating around `USE_CFP`. Thanks!